### ISSUE TITLE
Smithing bug?

### ISSUE THREAD

**Issue by Dooshken** [Guitarninja2/septimus#458]
Last night while I was playing, one smithing piece, a piece of base steel armor, had a +5 temper available. At the time I was at 59 smithing and had up to Journeyman in terms of perks, so that +5 seems to have been consistent. Then I made steel plate armor and +7 temper was available. I then leveled smithing a bit, up to 62 and +6 was available on the base steel armor, which still seems to be consistent. On the steel plate armor, it was +9. More specifically, I was making the steel spell knight armor, which is in the same class as steel plate, but I made steel plate armor to test it on and both were +9.

I did some further testing on silver equipment and the +6 was available on a silver weapon, without the dedicated silver smithing perk. I then took that perk and it went up to +9, like the steel plate stuff. I asked a few times and someone told me to report here, so hopefully this helps.

From more testing, it seems that at some point, the increase of +1 on some gear turns to +2 instead. I got up to 80 skill and the tempering I described before increased consistently, so now it's +7 on the base steel stuff and +11 on the steel plate stuff.

**Comment by Xgatt** [comment]
Are these what's available to you in order to temper, or do previously tempered items magically increase in temper amount as you start perking up?

**Comment by Dooshken** [comment]
> Are these what's available to you in order to temper, or do previously tempered items magically increase in temper amount as you start perking up?

Right now, it just seems that some armor types are getting +2 for every skill (20/40/60, etc) and perk increase instead of +1, from the Adamant plugin. I've started using dwarven and elven stuff since this post and they upgrade like the base steel armor, so seemingly consistent with the proper increase of +8 now. My smithing is over skill level 80 and I have the perk for ebony, stahlrim, etc.

But no, the increases over 10 so far have just been armors in the steel plate group, including the steel spell knight stuff, and silver equipment with the silver smithing perk. I don't have that perk, I just took it to test and it went over 10 consistently with the steel plate stuff.
